awscli/__init__.py

```diff
@@ -18,7 +18,7 @@
 
 import os
 
-__version__ = '1.35.6'
+__version__ = '1.35.8'
```

Comment:
This file should not be touched. You'll need to remove this from your commit. You can do this with some variation of:

```
git reset HEAD^ -- path/to/file
git commit --amend
```